web-src/src/components/PromptSessionSideView.l10n.js

```diff
@@ -142,5 +142,50 @@ export const intlMessages = {
       id: 'promptSessionSideView.audienceSelectorLoadTargetFailedToast',
       description: 'Toast message for failed Adobe Target loading',
     },
+    variationsGeneration15SecondsWaitTimeToast: {
```

Comment:
you need to generate localization for these messages. Please see how to do this https://adobe.enterprise.slack.com/canvas/C050XSQ2BCN?focus_section_id=temp:C:IFYd7c4bbc2ff914d2d9cdad63a4

Reply:
@irenelagno , thank you, I assumed they would be translated automatically

Reply:
@irenelagno , done, https://github.com/adobe/aem-genai-assistant/pull/274/commits/9dda0fb2be912c9d85dd191846ca8964dd749d49